### DISCUSSION TITLE
v0.23.0 - Extend with Go, unable to get the pocketbase@v0.23.0-rc (go mod init myapp && go mod tidy) to test

### DISCUSSION THREAD

**Discussion by MichaelSowah** [pocketbase/pocketbase#5630] [Q&A] [answered]
I wanted to test it with my extend with Go implementations, so I just created this to test and when I run go mod init myapp && go, I was still getting the v0.22.1 mod tidy
`package main

import (
    "log"
    "net/http"

    "github.com/pocketbase/pocketbase"
    "github.com/pocketbase/pocketbase/core"
)

func main() {
    app := pocketbase.New()

    app.OnServe().BindFunc(func(se *core.ServeEvent) error {
        // registers new "GET /hello" route
        se.Router.Get("/hello", func(re *core.RequestEvent) error {
            return re.String(200, "Hello world!")
        })

        return se.Next()
    })

    if err := app.Start(); err != nil {
        log.Fatal(err)
    }
}`

then I tried the below and this is the error I got
go install github.com/pocketbase/pocketbase@v0.23.0-rc
go: downloading github.com/pocketbase/pocketbase v0.23.0-rc
go: github.com/pocketbase/pocketbase@v0.23.0-rc: create zip: tests/data/aux.db: malformed file path "tests/data/aux.db": "aux" disallowed as path element component on Windows

Kindly assist, what am I doing wrong? I am on a Mac M1

**Comment by dschissler** [comment]
The first published module is `v0.23.0-rc2`, not `v0.23.0-rc`.

**Reply by MichaelSowah** [reply, marked as answer]
Yes, just noticed, fixed, came here to close the issue and saw your message, Thank you for the quick response. For any other person who stumbles here, I did `go get github.com/pocketbase/pocketbase@v0.23.0-rc2` and got it resolved

> The first published module is `v0.23.0-rc2`, not `v0.23.0-rc`.
